frontend/src/components/node/List.tsx

```diff
@@ -20,12 +21,6 @@ export default function NodeList() {
       resourceClass={Node}
       columns={[
         'name',
-        {
```

Comment:
I'm not sure about moving the Ready column to the end. I feel it's quite an important field, and if it's on the right side it can be hidden from view in some browser sizes.

What was the thinking for putting it at the end?

Reply:
I moved it a bit to frontend before taints and roles.

Reply:
Cool, thanks.